### ISSUE TITLE
outdated README examples

### ISSUE THREAD

**Issue by pbc** [jsmestad/pivotal-tracker#109]
Ruby 2.1.6, Rails 4.2, Mac OSX 10.10.3, gem version: 0.5.13

I think someone should have a look at the code examples in the README because they look heavily outdated.

For e.g. story object instantiation:

    PivotalTracker::Story.new(84739, :name => 'Ur Story')

which has incorrect number of params.

This code after fixing the instantiation:

    @a_project.stories << PivotalTracker::Story.new(:name => 'Ur Story')

will do nothing in terms association because the story won't be created via the API

On top of it when you run this bit of code next in IRB

    @a_project.stories

you will get this as an inspection result:

    PivotalTracker::Story

which is rather odd.

After that if you run this:

    @a_project.stories.all

you will get an empty array.

**Comment by jsmestad** [comment]
@pbc yeah I would not be surprised that documentation is outdated. If you have the time, mind putting together an MR to update it to working order?
